charts/ocis/docs/values-desc-table.adoc

```diff
@@ -1020,4 +1020,10 @@ a| [subs=-attributes]
 a| [subs=-attributes]
 `""`
 | URL to load themes from. Will be prepended to the theme path. Defaults to the value of "externalDomain".
+| topologySpreadConstraints
+a| [subs=-attributes]
++string+
+a| [subs=-attributes]
+`"- maxSkew: 1\n  topologyKey: kubernetes.io/hostname\n  whenUnsatisfiable: DoNotSchedule\n  labelSelector:\n    matchLabels:\n      app: {{ .appName }}\n"`
```

Comment:
Guessing that you want to have that renderd as in `values.adoc.yaml` below:
```
- maxSkew: 1
  topologyKey: kubernetes.io/hostname
  whenUnsatisfiable: DoNotSchedule
  labelSelector:
    matchLabels:
      app: {{ .appName }}
```
In case, we have to rewrite this a bit to make it work with antora 😄 
Let me know and I will make a suggestable commit.